.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,5 @@
 ## Checklist
 - [ ] This change requires a documentation update <!-- Update JIRA with Affects -> Documentation -->
 - [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added a test case that will be used to verify my changes 
```

Comment:
@davidffrench do you think will this line be enugh?

Reply:
@b1zzu Yes, and more importantly that it is enforced by the repo approvers on a PR